doc/developer_guide/aws.md

```diff
@@ -0,0 +1,160 @@
+
+| Name                 | Explanation                                                                               | Priority |
+|----------------------|-------------------------------------------------------------------------------------------|----------|
+| VMBucketRateLimit    | Declares the minimum possible rate limit for access: 100 requests in a 5 min interval.    | 0        |
+| CAPTCHA              | Forces a captcha action for any IP which does not match a predefined set of IP-addresses. | 1        |
+
+## Involved Cloudformation stacks
+
+The following diagram shows the involved cloudformation stacks:
+![image info](./img/cloudformation-stacks.drawio.png)
+
+The following resources are permanent and need to be deployed using the "deploy" [commands](#deployment-commands):
+* `DATA-SCIENCE-SANDBOX-VM-Bucket`
+* `DATA-SCIENCE-SANDBOX-CI-TEST-CODEBUILD`
+* `DATA-SCIENCE-SANDBOX-RELEASE-CODEBUILD`
+
+The EC2-stack lives only during the creation of a new sandbox image.
+
+## Tagging
+
+Each of the involved resources might cause costs: cloudformation stacks, AMI, EC2 key-pairs.
+
+To enable keeping track of all these resources, the implementation tags them after creation with a specific keyword (called __asset-id__).
+
+The S3 objects are identified by the prefix in the S3 bucket.
+
+The command tags only the dynamically created entities with the *asset-id* but not the permanent cloudformation stacks.
+
+The command `show-aws-assets` lists all assets which were created during the execution.
+* This is very useful if an error occured.
+* If the creation of a sandbox finished normally the list should contain only the AMI, images (S3 objects) and the export tasks (one for each image).
+
+## How to contribute
+
+The project has two types of CI tests:
+* unit tests and integration tests which run in a Github workflow
+* A system test which runs on a AWS Codebuild
+
+Both ci tests need to pass before the approval of a Github PR.
+The Github workflow will run on each push to a branch in the Github repository. However, the AWS Codebuild will only run after you push a commit containing the string "[CodeBuild]" in the commit message, see [Executing AWS CodeBuild](#executing-aws-codebuild).
+
+## Executing AWS CodeBuild
+
+Use the following git commands to execute the AWS CodeBuild script:
+```shell
+git commit -m "[CodeBuild]" --allow-empty && git push
+```
+
+This will trigger a webhook that was installed by an AWS template into the git-Repository.
+* The webhook is defined in file `exasol/ds/sandbox/templates/ci_code_build.jinja.yaml`
+* and calls `aws-code-build/ci/buildspec.yaml`
+* which then executes `test/codebuild/test_ci*.py`
+
+The CodeBuild will take about 20 minutes to complete.
```

Comment:
I propose to move sections "How to contribute" and "Executing AWS CodeBuild" to general developer_guide, too.